test/src/components/trends.spec.jsx

```diff
@@ -88,7 +88,7 @@ const stubSearchResults = [
             {value: 53, timestamp: 1508434548839}]
     },
     {
-        operationName: 'Op 2',
+        operationName: 'Op---2',
```

Comment:
I tried to test that the input had the correct operationName, but I was unable to figure out how to access this on the table row. It works when everything is deployed, but would prefer to have a unit test for spaces and periods in the operation name. Any help would be appreciated.